src/test/resources/org/opa/ideaplugin/lang/parser/fixtures/complex_rule_2.rego

```diff
@@ -11,7 +11,7 @@ warn[msg] {
     apiversions = ["apps/v1beta1", "apps/v1beta2"]
     resources =  as_array(input)
     r := resources[_]
-    not r.apiVersion == apiversions[_]
```

Comment:
what's wrong with not?

Reply:
The error when trying to execute it.

> 1 error occurred: policy.rego:14: rego_unsafe_var_error: var _ is unsafe

Not sure why it's not working but the way above do

https://play.openpolicyagent.org/p/RV1pwidP74